docs/2023/ABDM/Subscription-management/updates/2023-08-04.md

```diff
@@ -4,13 +4,15 @@ author: Suyash Kumar Dubey
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
+- [ ] Tested the subscription APIs on Postman after getting access. 
+- [ ] ABDM switched recently to new documentation, so I started reading those. A few errors were coming while testing some APIs, so I had to read the new documentation thoroughly and try it on the new ABDM app. 
+- [ ] Understood the workflow of implementing the facade with the existing APIs in the spring-boot application. 
```

Comment:
You are also supposed to mark as ticked for the ones which you completed.
Like this:
`- [x] Tested the subscription APIs on Postman after getting access.`

Reply:
I have added the tick marks in all my updates and committed them. Please check.